.github/actions/bump-version/action.yml

```diff
@@ -0,0 +1,149 @@
+name: Bump Version
+author: Tommy Gatti
+description: Action for bumping a version string
+inputs:
+  version:
+    required: true
+    type: string
+    description: The version string that will be bumped
+  versioning-scheme:
+    required: true
+    type: string
+    description: |
+      The type of versioning scheme used. Currently supports 'calver-minor', 'semver-major-minor', where:
+      'calver-minor': 'YYYY.OM.MINOR'.
+      'semver-major-minor': 'MAJOR.MINOR'.
+  bump-type:
+    required: true
+    type: string
+    description: |
+      The type of bump. This changes depending on the `versioning-scheme`.
+      'calver-minor': 'major' or 'minor'.
+      'semver-major-minor': 'major' or 'minor'.
+outputs:
+  before:
+    description: Version before being bumped. Equivalent to the input `version`.
+    value: ${{ steps.results.outputs.before }}
+  after:
+    description: Version after being bumped.
+    value: ${{ steps.results.outputs.after }}
+  type:
+    description: Type of bump. Equivalent to the input `bump-type`.
+    value: ${{ steps.results.outputs.type }}
+runs:
+  using: composite
+  steps:
+    - name: Check Versioning Scheme Validity
+      shell: bash
+      env:
+        VALID_VERSIONING_SCHEMES: calver-minor semver-major-minor
```

Comment:
Can we scope `VALID_VERSIONING_SCHEMES` to be "global" to be able to access it in another step?

Reply:
Done